decidim-system/app/forms/decidim/system/organization_form.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require "decidim/translatable_attributes"
+
+module Decidim
+  module System
+    # A form object used to update organizations from the system dashboard.
+    #
+    class OrganizationForm < Form
```

Comment:
Can we rename it to make it more clear? 
```suggestion
    class BaseOrganizationForm < Form
```